### ISSUE TITLE
How can I use VAAPI in SSR

### ISSUE THREAD

**Issue by wszqkzqk** [MaartenBaert/ssr#916]
Can SSR use VAAPI to encode the video just like OBS does?
![Z4KLY6T577BQ G49Z%CJEQJ](https://user-images.githubusercontent.com/12961135/135737598-9c2524b5-e5f6-4198-b4d9-877a4be36c01.png)

**Comment by mathieu-aubin** [comment]
do you have any vaapi installed?

`ffmpeg -hide_banner -encoders | grep vaapi`

**Comment by wszqkzqk** [comment]
> do you have any vaapi installed?
> 
> `ffmpeg -hide_banner -encoders | grep vaapi`

Yes, and I can use ffmpeg(encoded by hevc_vaapi) to record my screen in terminal

**Comment by wszqkzqk** [comment]
> do you have any vaapi installed?
> 
> `ffmpeg -hide_banner -encoders | grep vaapi`

Command like：
```
ffmpeg -vaapi_device /dev/dri/renderD128 -f x11grab -i :0 -f pulse -ac 2 -i 0 -vf 'hwupload,scale_vaapi=format=nv12' -c:a mp3 -c:v hevc_vaapi output.mp4
```
works on my device,but ssr still can't use VAAPI

**Comment by MaartenBaert** [comment]
SSR doesn't currently support VAAPI, and unfortunately it is not easy to add support for it.

**Comment by wszqkzqk** [comment]
> SSR doesn't currently support VAAPI, and unfortunately it is not easy to add support for it.

But why can it support AMF and QSV.

**Comment by mathieu-aubin** [comment]
https://trac.ffmpeg.org/wiki/Hardware/VAAPI

Couldnt SSR just change/update it's ffmpeg command line instruction?
